sheets/api.py

```diff
@@ -253,27 +253,41 @@ def update_spreadsheet_properties(self, file_id, property_dict):
             .execute()
         )
 
-    def get_sheet_properties(self, file_id):
+    def get_drive_file_metadata(self, file_id, fields="id, name, modifiedTime"):
```

Comment:
Would it be possible to have `fields` be a list of `str` and then convert it to a comma separated string in this function before the pygsheets function is called?

Reply:
I thought about that, but I'm leaning toward keeping it as is for now. Our use cases don't call for any list functionality yet (appending, checking members, etc.), it would take an extra line of code to set a default for these values, and we do have a use case where we just need a single field (so it's easier to pass in a simple string than a one-item list). I don't feel that strongly either way though. If you do, I'll make that change

Reply:
Ok, let's leave this as is